nextflow.config

```
@@ -1,56 +1,60 @@
 params {
 
-    outdir = "/hps/nobackup/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_v6"
-    dummy_fasta = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/dummy.fasta"
-    empty_file = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/EMPTY.txt"
+    outdir           = "/hps/nobackup/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_v6"
+    dummy_fasta      = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/dummy.fasta"
+    empty_file       = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/EMPTY.txt"
```

Hahaha these absolute paths I had written at first 😬 I have since learned a better way:
```suggestion
    dummy_fasta      = "$projectDir/assets/dummy.fasta"
    empty_file       = "$projectDir/assets/EMPTY.txt"
```